tests/cache/tests.py

```diff
@@ -92,10 +92,12 @@ def test_non_existent(self):
 
     def test_get_many(self):
         "get_many returns nothing for the dummy cache backend"
-        cache.set('a', 'a')
-        cache.set('b', 'b')
-        cache.set('c', 'c')
-        cache.set('d', 'd')
+        cache.set_many({
+            'a': 'a',
+            'b': 'b',
+            'c': 'c',
+            'd': 'd',
+        })
```

Comment:
Please collapse this down to a single line as it will easily fit in the 119 character line length limit.
And do so for all of the other changes in this PR.

Reply:
I shortened the code I wrote.